examples/hmc-agent/hmc_agent.container

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Run simple HMC AI agent application and MCP server
+Requires=hmc_config.service hmc_server.service
+After=hmc_config.service hmc_server.service
+
+[Service]
+Restart=on-failure
+RestartSec=60
+EnvironmentFile=/etc/pim/env.conf
+
+[Container]
+Image=na.artifactory.swg-devops.com/sys-pcloud-docker-local/devops/pim/hmc_agent
```

Comment:
let's push it inside app dir na.artifactory.swg-devops.com/sys-pcloud-docker-local/devops/pim/apps/hmc_agent